adaptations/Roland_SE02.py

```
@@ -0,0 +1,726 @@
+
+    This is the first concrete step toward USER-bank management:
+    - Editor bank exports encode slot in bb (05 bb cc dd).
+    - We retarget bb to `program_number` and recompute DT1 checksums.
+
+    IMPORTANT: This will overwrite the target slot on the synth if SE-02
+    accepts DT1 writes to program memory.
+    """
+    blob = _to_blob(message)
+    try:
+        target_bb = int(program_number) & 0x7F
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)